game.js

```diff
@@ -1,12 +1,25 @@
 const rls = require('readline-sync')
 
+//resolved my test problem
+
 /**
  * Starts the game by prompting the user if they want to play
  * Calls either gameLoop() or quitGame()
  * 
  * @returns {undefined}
  */
 const startGame = () => {
+  let args = process.argv.slice(2);
+  let guessCount = args[0];
+  let min = args[1];
+  let max = args[2];
```

Comment:
you have these variables that you never use throughout the rest of the code. You should delete anything you don't use